terraform/modules/jenkins/ec2.tf

```diff
@@ -0,0 +1,41 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_instance" "jenkins2" {
+  ami                    = "ami-785db401"
+  instance_type          = "t2.large"
+  iam_instance_profile   = "${aws_iam_instance_profile.jenkins.name}"
+  key_name               = "${aws_key_pair.jenkins2_2.key_name}"
+  vpc_security_group_ids = "${var.jenkins_security_group_ids}"
+
+  tags {
+    Name = "Jenkins2"
+  }
+}
+
+resource "aws_eip" "jenkins2" {
+  vpc = true
+}
+
+resource "aws_eip_association" "jenkins2_eip_assoc" {
+  instance_id   = "${aws_instance.jenkins2.id}"
+  allocation_id = "${aws_eip.jenkins2.id}"
+}
+
+resource "aws_key_pair" "jenkins2_2" {
```

Comment:
This key pair could probably have a better name. It was only called `jenkins2_2` originally because I mucked up the first key pair for the original `jenkins2`.

Reply:
Done - "jenkins" will probably do as a name, as it's now the same key as used for the current jenkins box. (We can always rotate in a new key later once the new box is the live box, but this keeps things simple in the creds repo.)